Cargo.toml

```diff
@@ -29,7 +29,7 @@ mozsvc-common = "0.1.0"
 num_cpus = "1.8.0"
 rand = "0.5.2"
 ring = "0.13.2"
-serde = "1.0.67"
+serde = { version = "1.0.67", features = ["rc"] }
```

Comment:
Feature added so that `Arc<ServerLimits>` can be serialized to a JSON response payload. The alternative is to derive `Clone` and `Copy` on ServerLimits and don't keep it in an `Arc`. I wasn't sure which would be preferable, happy to make the switch if this way is wrong.

Reply:
You should be able to get away w/out it. The inner ServerLimits can be deref'd out of the arc via `*server.limits` -- and we need to borrow it for the json call, so: `json(&*server.limits)`

Feel free to change this later if it makes it easier on these 2 outstanding PRs